Neos.Flow/Classes/Security/Policy/Role.php

```diff
@@ -19,7 +21,7 @@
  */
 class Role
 {
-    const ROLE_IDENTIFIER_PATTERN = '/^(\w+(?:\.\w+)*)\:(\w+)$/';   // Vendor(.Package)?:RoleName
+    protected const ROLE_IDENTIFIER_PATTERN = '/^(\w+(?:\.\w+)*)\:(\w+)$/';   // Vendor(.Package)?:RoleName
```

Comment:
I'd say. Let's make the class `final` and this constant `private`

Reply:
Devil's advocat: Making that class final would have to be considered breaking and be documented clearly. Let's maybe do this in #1939 instead?

Reply:
Yep, good point.